conda-store-server/Dockerfile

```
@@ -1,10 +1,20 @@
-FROM condaforge/mambaforge:23.3.1-1 as base
+# conda-store-server Dockerfile
+# the generated Docker image is used with docker-compose to run the conda-store
+# server and public conda-store Docker images
+
+FROM condaforge/mambaforge:24.3.0-0 AS base
```

Mambaforge is being [sunset](https://conda-forge.org/news/2024/07/29/sunsetting-mambaforge/), so maybe this should be:

```suggestion
FROM condaforge/miniforge:24.3.0-0 AS base
```

---

oooo I forgot, nice!